protocols/v1/src/json_rpc.rs

```diff
@@ -79,34 +77,3 @@ impl<'a> From<Notification> for Message {
         Message::Notification(n)
     }
 }
-
-struct DeserializeStringAndNumberIntoString;
```

Comment:
I was trying to leave it a string and add another one of these to convert the string to a number on the way out :) Your fix is cleaner though!